rclpy/rclpy/context.py

```diff
@@ -73,6 +85,7 @@ def shutdown(self):
         with self._handle as capsule, self._lock:
             rclpy_implementation.rclpy_shutdown(capsule)
         self._call_on_shutdown_callbacks()
+        self._logging_fini()
```

Comment:
@ivanpauno any idea why this is not called from `try_shutdown()` as well?

Reply:
no, I guess that was only a mistake.

Reply:
https://github.com/ros2/rclpy/pull/800 should fix it